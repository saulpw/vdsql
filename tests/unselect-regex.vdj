#!vd -p
{"longname": "open-file", "input": "tests/business.sqlite", "keystrokes": "o"}
{"sheet": "business", "col": "", "row": "\u30adcustomers", "longname": "open-row", "input": "", "keystrokes": "Enter", "comment": "open current row with sheet-specific dive"}
{"sheet": "customers", "col": "timezone", "row": "", "longname": "select-col-regex", "input": "Europe", "keystrokes": "|", "comment": "select rows matching regex in current column"}
{"sheet": "customers", "col": "timezone", "row": "", "longname": "unselect-col-regex", "input": "Minsk", "keystrokes": "\\", "comment": "unselect rows matching regex in current column"}
{"sheet": "customers", "col": "", "row": "", "longname": "dup-selected", "input": "", "keystrokes": "\"", "comment": "open duplicate sheet with selected rows (default limit)"}
