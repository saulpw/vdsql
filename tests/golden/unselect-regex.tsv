customerid	name	address	birthdate	phone	timezone	lat	long	locale
1008	高橋 くみ子	熊本県川崎市宮前区丹勢16丁目8番15号 コーポ鶴ヶ丘724	1966-08-14	38-8514-1974	Europe/Minsk	36.53333	136.61667	ja_JP
1018	Justin Morris	95274 Ricardo Pines
North Taylormouth, VA 06352	1984-05-11	597.052.2346x63522	Europe/Ljubljana	32.9156	-117.14392	en_US
1022	Maitê Rodrigues	Campo Nathan Cunha, 92
Vila Atila De Paiva
03920997 Cavalcanti do Campo / CE		(021) 9298 6896	Europe/Andorra	-12.91667	-39.25	pt_BR
1048	Ronald Ramirez	2858 Francisco Prairie Apt. 289
Port Brenda, ME 70743	1981-05-06	+1-121-296-3014x5029	Europe/Riga	38.06084	-97.92977	en_US
1056	Sherri Gonzalez	PSC 5573, Box 1138
APO AA 11320	2003-08-17	844-462-7381x6305	Europe/Amsterdam	29.65163	-82.32483	en_US
1060	Ana Beatriz Jesus	Colônia de da Mota, 16
Inconfidência
13740-031 da Cruz de da Rosa / TO	1973-11-23	(084) 7174 7621	Europe/Zagreb	-8.05389	-34.88111	pt_BR
1063	Richard Green	Unit 3733 Box 6473
DPO AA 82378	1965-10-01	095.243.7787x7493	Europe/Monaco	43.1009	-75.23266	en_US
1065	пані Єлисавета Цюцюра	шосе Вокзальний, буд. 36, Тальне, 28056	1998-01-13	063 70 83	Europe/Helsinki	49.65186	26.97253	uk_UA
1070	Устим Адамчук	вулиця Думська, буд. 55 кв. 592, Броди, 58888	1984-12-06	035 868-81-34	Europe/Kiev	49.65186	26.97253	uk_UA
